boards/riscv/longan_nano/longan_nano_defconfig

```diff
@@ -0,0 +1,18 @@
+# Sipeed longan nano Development Board Configuration
+#
+# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_SERIES_RISCV_GIGADEVICE_GD32VF103=y
+CONFIG_SOC_GD32VF103=y
+
+# 108MHz for system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=108000000
+
+# 27MHz for machine timer
+CONFIG_RISCV_MACHINE_TIMER=y
```

Comment:
I think no it isn't in theoretically.
But also I think it is not realistic because RISC-V currently not support single thread.
(because not set ARCH_HAS_SINGLE_THREAD_SUPPORT)

In other hand, all riscv soc not enable machine timer in soc level folder, but board level folder.
I think it is better that keep `CONFIG_RISCV_MACHINE_TIMER=y` line remain in board folder, at this time.